extra/modules/confiant-ad-quality/src/main/java/org/prebid/server/hooks/modules/com/confiant/adquality/core/BidsScanResult.java

```diff
@@ -25,11 +27,24 @@ public boolean hasIssues() {
                 .orElse(false);
     }
 
-    public List<BidderResponse> filterValidResponses(List<BidderResponse> responses) {
-        return IntStream.range(0, responses.size())
-                .filter(ind -> !hasIssuesByBidIndex(ind))
-                .mapToObj(responses::get)
-                .toList();
+    public Map<Boolean, List<BidderResponse>> toIssuesExistencyMap(List<BidderResponse> bidderResponses) {
```

Comment:
I'd suggest creating a separate return object for responses (with and without issues) in order to make it super clear, the method name could just `groupByIssues`

Another question: do the `public boolean hasIssues()` is still used?

Reply:
Thanks again. `groupByIssues` is already inside updates. 
Regarding `hasIssues` I had the same question to myself. But wanted to keep it for the future development and some tests scenarious. But if you have such question as well - means would be better to clean it now and add once we need that. So removed.